<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>373</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Reemplazo de p√°ginas</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblFrames">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Marcos:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblSequence">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Secuencia:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtFrames">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lblAlgorithms">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Algoritmo:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="cboAlgorithms"/>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QWidget" name="widgetSequence" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="txtSequence">
           <property name="text">
            <string>2,3,2,1,5,2,4,5,3,2,5,2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnRandom">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="res/resources.qrc">
             <normaloff>:/dice.png</normaloff>:/dice.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="QPushButton" name="btnExecute">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Ejecutar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="actionFileExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>A&amp;yuda</string>
    </property>
    <addaction name="actionHelpAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFileExit">
   <property name="icon">
    <iconset resource="res/resources.qrc">
     <normaloff>:/file-exit.png</normaloff>:/file-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Salir</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="icon">
    <iconset resource="res/resources.qrc">
     <normaloff>:/help-about.png</normaloff>:/help-about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Acerca de ...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnExecute</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotExecute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRandom</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotRandomSequence()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFileExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotFileExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHelpAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotHelpAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotExecute()</slot>
  <slot>slotRandomSequence()</slot>
  <slot>slotFileExit()</slot>
  <slot>slotHelpAbout()</slot>
 </slots>
</ui>
